pkg/chartutil/create.go

```diff
@@ -36,12 +36,37 @@ const (
 	ChartsDir = "charts"
 	// IgnorefileName is the name of the Helm ignore file.
 	IgnorefileName = ".helmignore"
+	// DeploymentName is the name of the example deployment file.
+	DeploymentName = "deployment.yaml"
+	// ServiceName is the name of the example service file.
+	ServiceName = "service.yaml"
```

Comment:
I'd prefer to use the "long form", I'd rather forward port these behaviors than backport ones from existing charts. svc vs service seems like an unnecessary pre-optimization.